worlds/sc2/Options.py

```diff
@@ -297,26 +303,10 @@ class GenericUpgradeItems(Choice):
     option_bundle_all = 3
 
 
-class NovaCovertOpsItems(Toggle):
-    """
-    If turned on, the equipment upgrades from Nova Covert Ops may be present in the world.
-
-    If Nova Covert Ops campaign is enabled, this option is locked to be turned on.
-    """
-    display_name = "Nova Covert Ops Items"
-    default = Toggle.option_true
-
-
-class BroodWarItems(Toggle):
-    """If turned on, returning items from StarCraft: Brood War may appear in the world."""
-    display_name = "Brood War Items"
-    default = Toggle.option_true
-
-
-class ExtendedItems(Toggle):
-    """If turned on, original items that did not appear in Campaign mode may appear in the world."""
-    display_name = "Extended Items"
-    default = Toggle.option_true
+class VanillaItemsOnly(Toggle):
+    """If turned on, the item pool is limited only to items that appear in the main 3 vanilla campaigns.
+    locked_items may override these exclusions."""
+    display_name = "Vanilla Items Only"
```

Comment:
How about NCO enabled? NCO isn't one of the 3 main campaigns, thus NCO items won't be present?
Surely, this option doesn't limit item availability cross campaign (like Vikings are still available in NCO)

Reply:
As I say in the comment, if players want more than the vanilla grouping, they should use item exclusions / item groups.

Exclude all terran units, unexclude NCO units. I'm not sure how we want to approach upgrades, WoL upgrades has a group, and could add NCO upgrades.